### PR TITLE
unify docker-compose files

### LINKED ISSUE #55: Unify docker compose yaml files

### PR DESCRIPTION
**Pull request by ldub**
Unifies the `docker-compose-*.yml` files and passes individual service names to docker-compose.

Closes #55

### CODE REVIEW

File: cli/cmd/start.go [hunk 1]
```diff
@@ -139,6 +139,11 @@ var images = map[string]bool{
 	"vulpemventures/nigiri-chopsticks:latest": true,
 }
 
+var composeServices = map[string]string{
+	"bitcoin": "bitcoin electrs esplora chopsticks",
+	"liquid": "electrs-liquid esplora-liquid chopsticks-liquid",
```
Comment (altafan) [inline cli/cmd/start.go]: missing `liquid` service?

File: cli/cmd/start.go [hunk 2]
```diff
@@ -265,6 +266,16 @@ func getStartBashCmd(datadir string) (*exec.Cmd, error) {
...
 
+func getComposeServices() (string, error) {
+	viper := config.Viper()
+
+	services := composeServices["bitcoin"]
+
+	if viper.GetBool("attachLiquid") {
+		services += " " + composeServices["liquid"]
+	}
+}
```
Comment (altafan) [inline cli/cmd/start.go]: missing return statement.
I think the function could return just `string` without `error`

**Comment by ldub** [comment]
Ok, this PR is hopelessly broken. I thought I tested it, but once again I was testing un-forked code to due how Go imports from GitHub work. I'll figure out a way to fix that permanently and then I'll fix this PR and update.

**Comment by tiero** [comment]
I think something around using new Go 1.11 modules instead of `dep` can be worth exploring

https://github.com/golang/go/wiki/Modules#example @ldub

**Pull request closed without merging**